README.md

```diff
@@ -7,3 +7,105 @@ This is a suitcase subpackage for reading a particular file format.
 ```
 pip install suitcase-jsonl
 ```
+
+## Common use cases.
+The common use case for this file format in streaming mode is to attach a
+RunRouter to the RunEngine (RE) using the following lines:
+
+```
+# create a factory that will use the serializer to create on-the-fly files.
+def factory(name, start_doc):
+    # the following creates a serializer that will close the file when the stop
+    # document is sent to it.
+    serializer=suitcase.jsonl.Serializer(directory='...',
+                                         file_prefix='...')
+    serializer(name, start_doc)
+    # ensure that any further documents from the RunRouter are processed.
+    return [serializer], []
+
+# generate the RunRouter from the factory and subscribe it too the RunEngine
+rr = event_model.RunRouter([factory])
+RE.subscribe(rr)
+```
+
+For a simple 'count' scan the file output will look like the following with
+each document type ('start', 'descriptor', 'event-page' or 'stop') found on a
```

Comment:
```suggestion
each document type found on a
```
The comprehensive list would also include event, datum, datum_page, and resource (where applicable). Maybe better just not to enumerate them here.